projects/packages/waf/src/class-brute-force-protection.php

```diff
@@ -515,10 +515,17 @@ public function get_protect_key() {
 	 *
 	 * Fires custom, plugable action jpp_log_failed_attempt with the IP
 	 *
-	 * @param string $login_user - the user attempting to log in.
+	 * @param string    $username - the user attempting to log in.
+	 * @param \WP_Error $error - the error object.
```

Comment:
```suggestion
	 * @param string    $username - The username or email address attempting to log in.
	 * @param \WP_Error $error    - A WP_Error object with the authentication failure details.
```
Copied in some extra details from param descriptions from the wp_login_failed core hook :+1: